src/types/onyx/Card.ts

```diff
@@ -154,6 +154,15 @@ type IssueNewCard = {
 
     /** Whether the user is editing step */
     isEditing: boolean;
+
+    /** Whether the request is being processed */
+    isLoading?: boolean;
+
+    /** Error message */
+    errors?: OnyxCommon.Errors;
+
+    /** Whether the request was successful */
+    success?: boolean;
```

Comment:
This should be`isSuccessful`

Reply:
@DylanDylann I saw "success" in a few places in the app like in https://github.com/Expensify/App/blob/d77a4fde57783ffb8d2443ae4394aecb8e55c4d4/src/types/onyx/Account.ts#L105 maybe it'd be better to stick to "success"

Reply:
We should follow our convention 

https://github.com/Expensify/App/blob/ce23fb55b1f8412d609cbd2c1f25690f7c5845a9/contributingGuides/STYLE.md?plain=1#L605-L622

Reply:
@DylanDylann shouldShowSuccess would be acceptable? 😅 like here https://github.com/Expensify/App/blob/ce23fb55b1f8412d609cbd2c1f25690f7c5845a9/src/types/onyx/WalletTransfer.ts#L17

Reply:
For me, I prefer `isSuccessful` because it's more generic